### ISSUE TITLE
Conda package

### ISSUE THREAD

**Issue by SimeonEhrig** [alpaka-group/alpaka#962]
As side product of the Ptychography 4.0 project, it is possible that we develop a Conda package for Aplaka, because we want to use Python and Alpaka together.

It can be useful, because it allows a easy install of Alpaka and Conda is more widely spread on Laptops than Spack, where people develop applications. 

Before we can start, the modernization of the cmake have to be finished: #942, #919

**Comment by ax3l** [comment]
Good idea, me and others took care that [pybind11](https://github.com/pybind/pybind11/) compiles with NVCC for follow-up steps.

[openPMD-api](https://github.com/openPMD/openPMD-api) should both demonstrate [conda packaging](https://github.com/conda-forge/openpmd-api-feedstock/) as well as pybind11 bindings.
